.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.761
+    hooks:
+      - id: mypy
```

Comment:
Hey, just checking - this now checks all python files with mypy?

Reply:
Also, once we know the overall setup is working, can we make this as strict as possible, i.e. no `Any` and requiring everything to be typed?